components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -963,6 +965,21 @@ private static void setConsentRequiredScopesToOAuthParams(AuthenticatedUser user
                 }
             }
             if (CollectionUtils.isNotEmpty(allowedOAuthScopes)) {
+                // Filter out internal scopes to be validated.
+
```

Comment:
remove extra newline

Reply:
This has been fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1766/commits/e16d6d7f40982b7d4d021dfc9e0e3ff2358e8835